docs/api/index.md

```diff
@@ -906,3 +910,92 @@ afterAll(async () => {
 ```
 
 Here the `afterAll` ensures that `stopMocking` method is called after all tests run.
+
+## Test Hooks
+
+Vitest provides a few hooks that you can call _during_ the test execution to cleanup the state when the test has finished runnning.
+
+::: warning
+These hooks will throw an error if they are called outside of the test body.
+:::
+
+### onTestFinished <Badge type="info">1.3.0+</Badge>
+
+This hook is called after the test has finished running. It is called after `afterEach` hooks since they can infulence the test result. It receives a `TaskResult` object with the current test result.
```

Comment:
```suggestion
This hook is called after the test has finished running. It is called after `afterEach` hooks since they can influence the test result. It receives a `TaskResult` object with the current test result.
```